templates/default/web_app.conf.erb

```diff
@@ -3,10 +3,10 @@
 #  Local modifications will be overwritten by Chef.
 #
 <VirtualHost *:<%= node['stash']['apache2']['port'] %>>
-	<% unless node['stash']['apache2']['virtual_host_name'].empty? -%>
```

Comment:
Hm. Seems we *are* using blank strings for some of this stuff in attrs... thoughts? Using `nil` for unset things seems more sensible to me. Or we could mix-and-match approaches in this template, and just assume nil is the blank value for the `virtual_host_*` attrs.